core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -300,7 +300,7 @@ class PluginTypeChecker(platform: String = "onprem") extends Logging {
     val exprLowercase = if (expr.equalsIgnoreCase("date_format")) {
       expr
     } else {
-      expr.toLowerCase.replace("_", "")
+      expr.toLowerCase
```

Comment:
looks like the new change deprecate the `if-block`.
We can get rid of the redundant condition because I cannot find we use the upper_case of `date_format` in the `supportedExprs.csv` file

```
val exprLowercase = expr.toLowerCase
```

Reply:
Thanks! Updated it.